src/modules/ethereum/transactions/interpret.ts

```diff
@@ -20,12 +20,19 @@ export function interpretTransaction({
   client?: Client;
   currency: string;
 }): Promise<InterpretResponse> {
-  const gas = getGas(transaction);
   return Promise.race([
     rejectAfterDelay(10000, 'interpret transaction'),
     new Promise<InterpretResponse>((resolve) => {
       let value: InterpretResponse | null = null;
 
+      const normalizedTx = { ...transaction };
```

Comment:
maxFee and maxFeePerGas had different names in transaction and interpret endpoint previously. Is this ok?